pontoon/base/static/js/translate.js

```diff
@@ -236,7 +236,7 @@ var Pontoon = (function (my) {
                       '<time class="stress" datetime="' + this.date_iso + '">' + this.date + '</time>' +
                     '</div>' +
                     '<menu class="toolbar">' +
-                      '<button class="approve fa" title="' +
+                      '<button class="' + (this.approved ? 'unapprove' : 'approve') + ' fa" title="' +
                       (this.approved ? this.approved_user ?
                         'Approved by ' + this.approved_user : '' : 'Approve') +
```

Comment:
The title tag should be changed to `approve`/`unapprove`. `Approved by approver` can be moved to the `title` tag of the translation author (line 235).